spec/models/user_spec.rb

```diff
@@ -71,14 +71,6 @@
       it { is_expected.not_to allow_value('invali-email.com').for(:email) }
     end
 
-    it 'is invalid with an image size greater 2mb' do
```

Comment:
why you remove this spec?

Reply:
I had problems with this test due to a fictitious image, but I have already corrected and returned the test